SharedSource/Development/Moq/UnitTesting/MoqExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.  You may obtain a copy of the
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+
+using System;
+using Moq.Language;
+using Moq.Language.Flow;
+using Remotion.Utilities;
+
+namespace Remotion.Development.Moq.UnitTesting
+{
+  public static partial class MoqExtensions
```

Comment:
The Extension method could be a separate ticket/PR to keep things separated. do we have time for this?

Reply:
yes, I will do it right now

Reply:
https://github.com/re-motion/Framework/pull/155